src/support/z_context.erl

```diff
@@ -400,7 +401,7 @@ prune_reqdata(ReqData) ->
 %% @spec abs_url(iolist(), Context) -> binary()
 abs_url(<<"http:", _/binary>> = Url, _Context) ->
     Url;
-abs_url(<<"https", _/binary>> = Url, _Context) ->
+abs_url(<<"https:", _/binary>> = Url, _Context) ->
     Url;
 abs_url(<<"//", _/binary>> = Url, Context) ->
     <<(site_protocol(Context))/binary, $:, Url/binary>>;
```

Comment:
I wonder if `abs_url` should sanitize. In its use elsewhere at least in some cases no sanitization is done, or am I overlooking something? E.g. line 443 `site_url("/" ++ _ = Path, Context) -> abs_url(Path, Context);`

Reply:
I think the input should be sanitized by the caller, we use this for different formatted URLs (hex encoded, HTML escaped, XML escaped).